pillarbox-demo/src/main/AndroidManifest.xml

```diff
@@ -28,6 +36,37 @@
             android:exported="true"
             android:launchMode="singleTask"
             android:supportsPictureInPicture="true" />
+        <activity
+            android:name=".ui.player.mediacontroller.MediaControllerActivity"
+            android:configChanges="orientation|screenSize|screenLayout|keyboardHidden"
+            android:exported="true"
+            android:launchMode="singleTask"
+            android:supportsPictureInPicture="true" />
+
+        <service
+            android:name=".service.DemoPlaybackService"
+            android:foregroundServiceType="mediaPlayback" />
+
+
+        <service
+            android:name=".service.DemoMediaSessionService"
+            android:exported="true"
+            android:foregroundServiceType="mediaPlayback">
+            <intent-filter>
+                <action android:name="androidx.media3.session.MediaSessionService" />
+            </intent-filter>
+        </service>
+
+        <service
```

Comment:
⚠️ **[Android Lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>ExportedService: Exported service does not require permission